helm/presets/local.yaml

```diff
@@ -28,9 +28,6 @@ strimzi:
       limits:
         cpu: 100m
         memory: 512Mi
-redis:
```

Comment:
same question: if we want to leave this in and just set it to `enabled: false` the  user can choose redis or postgres just by the configuration in their `releases.exs`